docs/quickstarts/ruby-quickstart.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: docs
+title: Ruby Quickstart
+---
+
+<h1 class="page-header">Ruby Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in Ruby with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+ * `ruby`: version 2 or higher
+
+### Install gRPC
+
+```
+  $ gem install grpc
+```
+
+### Install gRPC tools
+
+Ruby's gRPC tools include the protocol buffer compiler `protoc`
+and the special plugin for generating server and client code
+from the `.proto` service definitions. For the first part of our
+quickstart example, we've already generated the server and client
+stubs from [helloworld.proto](https://github.com/grpc/grpc/tree/{{site.data.config.grpc_release_branch}}/examples/protos/helloworld.proto),
+but you'll need the tools for the rest of our quickstart, as well as later
+tutorials and your own projects.
+
+To install gRPC tools, run:
+
+```sh
+gem install grpc-tools
+```
+
+## Download the example
+
+You'll need a local copy of the example code to work through this quickstart. Download the example code from our Github repository (the following command clones the entire repository, but you just need the examples for this quickstart and other tutorials):
+
+```sh
+  $ # Clone the repository to get the example code:
+  $ git clone https://github.com/grpc/grpc
+  $ # Navigate to the "hello, world" Ruby example:
+  $ cd grpc/examples/ruby
+```
+
+## Run a gRPC application
+
+From the `examples/python` directory:
+
+1. Run the server
+
+   ```sh
+   $ ruby greeter_server.rb
+   ```
+
+2. In another terminal, run the client
+
+   ```sh
+   $ ruby greeter_client.rb
+   ```
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the
+server for the client to call. Our gRPC service is defined using protocol
+buffers; you can find out lots more about how to define a service in a `.proto`
+file in [What is gRPC?]() and [gRPC Basics: Ruby][]. For now all you need
```

Comment:
Discussed with @zinuga: We can drop the reference to What is gRPC? here to simplify.